frontend/src/pages/TermPlanner/TermPlanner.tsx

```diff
@@ -272,7 +272,7 @@ const TermPlanner = () => {
           // TODO: Fix Suspense by updating to react-query v5
           /* <Suspense fallback={<Spinner text="Loading Table..." />}> */
         }
-        {courseQueries.some((c) => c.isPending) || !courseInfos[LIVE_YEAR] ? (
+        {courseQueries.some((c) => c.isPending || coursesQuery.isPending || plannerQuery.isPending )  ? (
```

Comment:
What motivated this change? Also, I don't think the added content should be inside the call to `some`.

Reply:
good spotting. i'll fix this and send a pr again